Sources/Commands/Workspace.swift

```diff
@@ -218,11 +218,11 @@ public class Workspace {
         }
 
         /// Find a manifest given its name.
-        func lookup(_ name: String) -> Manifest? {
-            return dependencies.first(where: { $0.name == name })
+        func lookup(_ name: String) -> (manifest: Manifest, dependency: ManagedDependency)? {
```

Comment:
How many clients want both? Would having `lookup(manifest name: String)` and `lookup(package name: String) make more sense (rather than unit test changes which just add a bunch of `.manifest`?